### DISCUSSION TITLE
How to create a custom carousel?

### DISCUSSION THREAD

**Discussion by AndrasE** [igordanchenko/yet-another-react-lightbox#295] [Q&A] [answered]
I'm trying to create a simple custom carousel (not opening the lightbox in full, just a bland carousel) that will display testimonals instead of just images. 

I have a few of these objects that I intend to map through to be displayed. 

```js
export const testimonialsData = [
  {
    src: require("../../../images/1.png"),
    name: "Andras",
    rating: "⭐⭐⭐⭐⭐",
    review: "Yet and other but the bestest lightbox",
  },..
```

The star wars example was great, but I cannot seems to make it work. I use jsx not tsx. Thank you for anyone you could give me some pointers!

**Comment by igordanchenko** [comment, marked as answer]
You can render additional elements with the `slideFooter` render function and use `position: absolute`.

https://yet-another-react-lightbox.com/documentation#Render

If you do not want to use absolute positioning, then you'll need to use `slide` render function and render image and all additional elements yourself.

**Reply by AndrasE** [reply]
Thank you again Igor! Ill check it out!
